web/_js/atlas.js

```diff
@@ -517,7 +517,8 @@ var atlas = [
 {"id": 467, "name": "MechWarrior Timber Wolf (Mad Cat)", "description": "A depiction of a Timber Wolf (Mad Cat) from MechWarrior and BattleTech.\n\nBased on the .exe icon for MechWarrior 3, and the explosion background on the MechWarrior 2 box art.", "website": "", "subreddit": "/r/MechWarrior", "center": [1348.5, 533.5], "path": [[1339.5, 522.5], [1339.5, 543.5], [1357.5, 543.5], [1357.5, 522.5]] },
 {"id": 468, "name": "Genshin Impact Icon","description": "The Genshin Impact Icon, a gatcha game made by hoyoverse","website": "","subreddit": "r/GenshinPlace","center": [1750.5,1787.5],"path": [[1737.5,1762.5],[1771.5,1762.5],[1772.5,1810.5],[1723.5,1808.5],[1727.5,1802.5],[1728.5,1801.5],[1729.5,1800.5],[1730.5,1798.5],[1731.5,1797.5],[1731.5,1793.5],[1730.5,1792.5],[1729.5,1791.5],[1728.5,1791.5],[1728.5,1790.5],[1727.5,1790.5],[1726.5,1789.5],[1725.5,1788.5],[1724.5,1788.5],[1724.5,1774.5],[1737.5,1774.5],[1737.5,1771.5]]},
 {"id": 469, "name":"Modrinth","description":"The logo for Modrinth, the open source modding platform.","website":"https://modrinth.com","subreddit":"","center":[730.5,642.5],"path":[[727.5,646.5],[727.5,645.5],[726.5,645.5],[726.5,644.5],[725.5,644.5],[725.5,639.5],[726.5,639.5],[726.5,638.5],[727.5,638.5],[727.5,637.5],[732.5,637.5],[732.5,638.5],[733.5,638.5],[733.5,639.5],[734.5,639.5],[734.5,644.5],[733.5,644.5],[733.5,645.5],[732.5,645.5],[732.5,646.5],[727.5,646.5]]},
-{"id": 470, "name": "Gem Galaxies", "description": "Gem Galaxies is a game on ROBLOX inspired by Steven Universe that is dedicated to roleplaying: the symbol that was created is of their diamond insignia.", "website": "https://www.roblox.com/games/273326777/Gem-Galaxies", "subreddit": "", "center": [ 1255.5, 897.5 ], "path": [ [ 1255.5, 891.5 ], [ 1250.5, 898.5 ], [ 1251.5, 899.5 ], [ 1255.5, 900.5 ], [ 1261.5, 899.5 ], [ 1256.5, 892.5 ] ] }
+{"id": 470, "name": "Gem Galaxies", "description": "Gem Galaxies is a game on ROBLOX inspired by Steven Universe that is dedicated to roleplaying: the symbol that was created is of their diamond insignia.", "website": "https://www.roblox.com/games/273326777/Gem-Galaxies", "subreddit": "", "center": [ [ 1255.5, 897.5 ], "path": [ 1255.5, 891.5 ], [ 1250.5, 898.5 ], [ 1251.5, 899.5 ], [ 1255.5, 900.5 ], [ 1261.5, 899.5 ], [ 1256.5, 892.5 ] ] },
```

Comment:
```suggestion
{"id": 470, "name": "Gem Galaxies", "description": "Gem Galaxies is a game on ROBLOX inspired by Steven Universe that is dedicated to roleplaying: the symbol that was created is of their diamond insignia.", "website": "https://www.roblox.com/games/273326777/Gem-Galaxies", "subreddit": "", "center":  [ 1255.5, 897.5 ], "path": [ [ 1255.5, 891.5 ], [ 1250.5, 898.5 ], [ 1251.5, 899.5 ], [ 1255.5, 900.5 ], [ 1261.5, 899.5 ], [ 1256.5, 892.5 ] ] },
```
If you didn't notice, you modified Gem Galaxies which resulted in your PR not working.
You can view the preview [here](https://deploy-preview-335--place-atlas.netlify.app/), but yeah, it's loading infinity.
After this it's hopefully fixed? Sorry for the many reviews

Reply:
No problem, thanks for letting me know!